<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QDialog" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>167</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>167</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>167</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>База данных:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLogin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>postgres</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>energo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
